frontend/src/reducer/currentWorkspace.js

```diff
@@ -160,7 +160,7 @@ export default function currentWorkspace (state = defaultWorkspace, action) {
         ...state,
         contentReadStatusList: state.contentReadStatusList.filter(id => id !== action.unreadId),
         recentActivityList: [
-          ...(state.recentActivityList.find(content => content.id === action.unreadId) || []),
+          ...state.recentActivityList.filter(content => content.id === action.unreadId),
```

Comment:
I'm confused, this seems to mean that recentActivityList can have the same contend id multiple times. I don't think it does

Reply:
So I don't understand the purpose of this

Reply:
The purpose was to return an empty array if the content is not in the `recentActivityList` instead of `null`
But I removed it and fix the problem with adding reducer in `currentWorkspace` for share content